profile/README.md

```diff
@@ -1,69 +1,13 @@
-# Hi, welcome to Å Energi @ GitHub!
+# 👋 Hi, welcome to Å Energi @ GitHub!
+Here at Å Energi we use GitHub to create some really cool stuff. But sadly we don't have that much to show right here on GitHub. But go to [developer.aenergi.no](https://developer.aenergi.no/) if you want to browse some more 😉
 
-It's a pleasure to have you join our organisation on GitHub. We have a few important things you should check out before you continue, to ensure that we all work well together.
+## 🔐 Want access?
+Are you an Å Energi employee that want access to our GitHub community and all the sweet stuff? 🍭
```

Comment:
Hva med Glitre Nett, Å Insite og andre datterselskaper, assosierte selskaper og konsulenter? Skal vi skrive noe om "or if you are associated with Å Energi" eller noe i den duren?

Reply:
Omformulerte litt nå for å få med en liten liste datterselskaper 💪